cmd/groupsio/groupsio.go

```diff
@@ -166,30 +166,26 @@ func (j *DSGroupsio) ParseArgs(ctx *shared.Ctx) error {
 		j.GroupName = ctx.Env("GROUP_NAME")
 	}
 	// Email
-	if shared.FlagPassed(ctx, "email") && *j.FlagEmail != "" {
+	if (shared.FlagPassed(ctx, "email") && *j.FlagEmail != "") || ctx.EnvSet("EMAIL") {
 		j.Email, err = encrypt.Decrypt(*j.FlagEmail)
```

Comment:
Actually, this won't work - you have an if for either flag or env - but then (inside if) you assume that flag was used).
I think you should first get a value from either flag or env, and then decrypt it if not empty. You can close thsi PR because I'm going to update all connectors myself.

Reply:
@lukaszgryglicki Sure, closing this PR